docs/hub/mlx-image.md

```diff
@@ -0,0 +1,89 @@
+# Using mlxim at Hugging Face
+
+[`mlxim`](https://github.com/riccardomusmeci/mlx-image) is an image models library built on Apple [MLX](https://github.com/ml-explore/mlx). It tries to replicate the great [timm](https://github.com/huggingface/pytorch-image-models) library from Ross Wightman, but for MLX models.
+
+
+## Exploring MLX on the Hub
+
+You can find `mlxim` models by filtering at the left of the [models page](https://huggingface.co/models?library=mlxim&sort=trending).
+There's also an open [mlx-vision](https://huggingface.co/mlx-vision) space for contributors converting and publishing weights for MLX format.
+
+Thanks to MLX Hugging Face Hub integration, you can load MLX models with a few lines of code. 
+
+## Installation
+
+```bash
+pip install mlx-image
+```
+## **Models**
+Models weights are available on [`mlx-vision`](https://huggingface.co/mlx-vision) space on HuggingFace.
+
+To create a model with weights:
+```python
+from mlxim.model import create_model
+
+# loading weights from HuggingFace
+model = create_model("resnet18")
```

Comment:
Interesting! What repo are the weights coming from? Can we override?

Reply:
Override HF weights is not possible currently, but it's a good point for the future. I currently assume to download weights from the defined mlx-vision hubs, but one can load the weights from his/her space. Working on it.

Reply:
I didn't mean this was a requirement to release the docs! It's cool if you've already done it, otherwise we can just state where the weights are downloaded from.